bindings/ios/MEGASdk.h

```diff
@@ -161,6 +161,19 @@ typedef NS_ENUM(NSUInteger, StorageState) {
     StorageStateChange = 3
 };
 
+typedef NS_ENUM(NSInteger, SMSState) {
+    SMSStateNotAllowed = 0,
+    SMSStateOnlyUnblock = 1,
+    SMSStateOptInAndUnblock = 2,
+};
+
+typedef NS_ENUM(NSInteger, AccountSuspensionType) {
+    AccountSuspensionTypeNone = 0, // The account is not blocked
+    AccountSuspensionTypeNonCopyright = 200, // suspension for any type of suspension, but copyright suspension
+    AccountSuspensionTypeCopyright = 300, // suspension only for multiple copyright violations
```

Comment:
Very likely, the PR https://github.com/meganz/sdk/pull/1463 will get merged sooner than this one. Perhaps you want to add the new reasons to the enumeration at that time: https://github.com/meganz/sdk/pull/1463/files#diff-c3de3d92d9d9472cc3d715cb978d2940R13851

Reply:
Not sure I understand. How about just resolve the conflicts when they occur?

Reply:
Hi Sergio, the enum is created in iOS binding project to provide support to iOS code. This is to avoid magic numbers in iOS code. Yes, we need to add the new suspension types (400, 401) to the list later on. Thanks for your reminder.